controllers/schedule_controller.go

```diff
@@ -66,7 +66,7 @@ type BackupScheduleReconciler struct {
 //+kubebuilder:rbac:groups=cluster.open-cluster-management.io,resources=backupschedules/status,verbs=get;update;patch
 //+kubebuilder:rbac:groups=cluster.open-cluster-management.io,resources=backupschedules/finalizers,verbs=update
 //+kubebuilder:rbac:groups=cluster.open-cluster-management.io,resources=managedclusters,verbs=get;list;watch
-//+kubebuilder:rbac:groups=cluster.open-cluster-management.io,resources=channels,verbs=get;list;watch
+//+kubebuilder:rbac:groups=apps.open-cluster-management.io,resources=channels,verbs=get;list;watch
 //+kubebuilder:rbac:groups=velero.io,resources=schedules,verbs=get;list;watch;create;update;patch
```

Comment:
Thanks for the fix :)

Reply:
I would expect an update in `role.yaml` file, did you run `make manifests`?